api/app/weather_models/__init__.py

```diff
@@ -1 +1,33 @@
-"""Code common to app.weather_models"""
+"""Code common to app.weather_models.fetch"""
+
+from enum import Enum
+
+# Key values on ModelRunGridSubsetPrediction.
+# Wind direction (wdir_tgl_10_b) is handled slightly differently, so not included here.
+SCALAR_MODEL_VALUE_KEYS = ("tmp_tgl_2", "rh_tgl_2", "wind_tgl_10")
```

Comment:
Should we put this stuff in `wps_shared` and import it from there instead of declaring it here and in `wps_jobs`?